src/homepage/_posts/2021-03-16-james-3.6.0.markdown

```diff
@@ -0,0 +1,165 @@
+- JAMES-3212 JMAP Handle subcrible/unsubcrible child's folder when update mailbox
+- JAMES-3416 Fix ElasticSearch email address search
+- JAMES-1677 Upgrade default hasing algorithm to SHA-512
+- JAMES-3454 Use a callback mechanism to re-create RabbitMQ auto-delete queues upon reconnections
+- JAMES-3296 Recover email sent during RabbitMQ outages
+- JAMES-2046 SentDateComparator should fallback to Mimle4J parsers
+- JAMES-3416 ElasticSearch address indexing fixes
+- JAMES-3386 add test to ensure blank mailbox paths are not allowed in jmap draft
+- MAILBOX-392 WebAdmin documentation: creation of mailboxes with '&' is allowed
+- JAMES-3380 use non am/pm dependent hour format
+- JAMES-2220 JMAP Draft: Flags update should not fail when a user is missing its Outbox
+- JAMES-3364 DeletedMessageVault: deleting many messages dead-locks
+- JAMES-3361 JMAP Draft: sharee should not be able to modify mailbox rights
+- JAMES-3308 RabbitMQTerminationSubscriberTest should be thread safe
+- JAMES-3177 Applicable flags updates needs to be thread safe (IMAP SELECT)
+- JAMES-3309 Avoid a NPE in FetchProcessor when SelectedMailbox is unselected
+- JAMES-3300 Fix default Cassandra LDAP configuration
+- JAMES-3267 Stop forcefully delete ImapRequestFrameDecoder.decode temporary file
+- JAMES-3167 Reactify MailboxMapper - unlocks better concurrency management
+- JAMES-3170 Fix metric measurement upon reactor publisher replay
+- JAMES-3213 Source ReplyTo in ICALToJsonAttribute
+- JAMES-3204 Push limit to Cassandra backend when reading messages - before that message listing queries where always reading at least 5000 rows, and triggering other reads for these rows.
+- JAMES-3201 ReIndexing enhancements
+- JAMES-3179 Fix UpdatableTickingClock thread safety issue
+- MAILBOX-405 Renaming too much mailboxes at once was failing on top of the Cassandra mailbox
+- JAMES-3513 Wrong UID dispatched on the EventBus for StoreMessageIdManager::setInMailboxes
+- JAMES-3512 DigestUtil: close base64 encoding stream
+- JAMES-3487 Allow setting on*Exception parameters for Bounce
+- JAMES-3511 Solve java.util.NoSuchElementException: heartbeatHandler
+- JAMES-3507 Fix broken IMAP APPEND literalSizeLimit option preventing from buffering large requests to files
+- JAMES-3438 des-ambiguity error message for Email/set create Content-Transfer-Encoding rejection
+- JAMES-3477 Fix NPE when concurrently updating MimeMessage
+- JAMES-3444 Perform JMAP TransportChecks only when JMAP is enabled
+- JAMES-3495 Cassandra mailbox: Reproduce and fix the null messageId bug
+- JAMES-3490 maxUploadSize should come from configuration
+- JAMES-1717 VacationMailet should not return answers when no or empty Reply-To header
+- JAMES-1784 JMAP: Users with `_` in their names cannot download attachments
+
+### Removed
+- HybridBlobStore. This will be removed after 3.6.0 release. Introduced to fasten small blob access, its usage could be
```

Comment:
hmm so is it deprecated or removed ? :)

Reply:
It had been deprecated in 3.5.0 and removed before 3.6.0. I will clarify this.